CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2232**
   - Updated versions for `ajv`, `lodash`, `googleapis`, `archiver`, and `@cumulus/aws-client` to remediate vulnerabilities found in SNYK scan.
 
+### Fixed
+- **CUMULUS-2233**
+  - Fixes /s3credentials bug where user ends up in redirect loop because the expirationTime on the access token generated by EarthdataLoginClient is in seconds and Date time expects milliseconds. Fixed by converting seconds into milliseconds. Unit tests were added to test that the expiration time has been converted to milliseconds.
```

Comment:
I think we need to clarify that the real "bug" here from a user perspective was that we were setting an expired cookie, so authentication was never being recognized as complete by the API. The information about seconds/milliseconds is the underlying cause of that bug, which is fine to leave as long as we clarify the actual bug being fixed.

Also, I think what our unit tests **should** be proving is that the cookie is not expired if a non-expired access token response is received, assuming they don't already prove that